ci/meta.yml

```diff
@@ -1,22 +1,25 @@
 meta:
   plan:
-    terraform-output:
-      task: terraform-output
+    terraform-common-config:
       config:
         platform: linux
         image_resource:
           type: docker-image
           source:
-            repository: ((terraform.repository))
-            version: ((terraform12.version))
-            tag: ((terraform12.version))
+            repository: ((dataworks.terraform_repository))
+            version: ((dataworks.terraform_version))
+            tag: ((dataworks.terraform_version))
         params:
-          AWS_ACCESS_KEY_ID: ((ci.aws_access_key_id))
-          AWS_SECRET_ACCESS_KEY: ((ci.aws_secret_access_key))
           TF_INPUT: false
           TF_CLI_ARGS_apply: -lock-timeout=300s
-          TF_CLI_ARGS_plan:  -lock-timeout=300s
-          TF_VAR_slack_webhook_url: ((slack.webhook_url))
+          TF_CLI_ARGS_plan: -lock-timeout=300s
+          TF_VAR_slack_webhook_url: ((dataworks.slack_webhook_url))
+
+    terraform-output:
+      task: terraform-output
+      .: (( inject meta.plan.terraform-common-config ))
```

Comment:
if this is only injected once then it's not being reused so just obscuring the config

Reply:
This is a pattern we use elsewhere and if any other tasks were to be added later this would help.